lib/notifier/index.js

```diff
@@ -246,6 +246,41 @@ function sendTransactionMessage (rec, isHighValueTx) {
   })
 }
 
+function cashboxNotify (deviceId) {
+  return Promise.all([
+    settingsLoader.loadLatest(),
+    queries.getMachineName(deviceId)
+  ])
+    .then(([settings, machineName]) => {
+      const notifications = configManager.getGlobalNotifications(settings.config)
+      const rec = {
+        sms: {
+          body: `Cashbox removed - ${machineName}`
+        },
+        email: {
+          subject: `Cashbox removal`,
+          body: `Cashbox removed in machine ${machineName}`
+        }
+      }
+
+      const promises = []
+
+      const emailActive =
+        notifications.email.active &&
+        notifications.email.security
+
+      const smsActive =
+        notifications.sms.active &&
+        notifications.sms.security
+      
+      if (emailActive) promises.push(emailFuncs.sendMessage(settings, rec))
```

Comment:
This notification logic should be general purpose for all notifications. @chaotixkilla can you create a new Trello card to generalize the logic?